regen/feature.pl

```diff
@@ -761,6 +762,18 @@ =head2 The 'isa' feature
 
 This feature is available from Perl 5.32 onwards.
 
+=head2 The 'indirect' feature
+
+This feature allows the use of L<indirect object
+syntax|perlobj/Indirect Object Syntax> for method calls, e.g.  C<new
+Foo 1, 2;>. It is enabled by default, but can be turned off to
+disallow indirect object syntax.
+
+This feature is available under this name from Perl 5.32 onwards. In
+previous versions, it was simply on all the time.  To disallow (or
+warn on) indirect object syntax on older Perls, see the L<indirect>
+CPAN module.
+
```

Comment:
Given the comments I had on the original branch, this should probably mention that disabling it has no effect on the similar syntax for print, exec etc.